src/Illuminate/Foundation/helpers.php

```diff
@@ -110,7 +110,7 @@ function action($name, $parameters = [], $absolute = true)
      *
      * @param  string|null  $abstract
      * @param  array  $parameters
-     * @return mixed|\Illuminate\Contracts\Foundation\Application
```

Comment:
While I agree with this change, people are going to complain it messes up IDE autocompletion if you remove this.

Reply:
Okay, I see. Should I put it back then?

Reply:
Yes, I think so.